mantidimaging/core/io/loader/img_loader.py

```diff
@@ -52,7 +52,7 @@ def execute(load_func: Callable[[str], np.ndarray],
 
     sample_data = il.load_sample_data(chosen_input_filenames)
 
-    return ImageStack(sample_data, chosen_input_filenames, indices)
+    return ImageStack(sample_data, chosen_input_filenames, indices, projection=True)
```

Comment:
I think we go through this path for all stacks that are loaded from a file, including flat and dark frames.

Its ok for now, but eventually setting an ImageStack as having a geometry will need to be somewhere like `MainWindowModel.do_load_dataset()`.

Reply:
Thanks